### PR TITLE
code_of_conduct addded

### PR DESCRIPTION
**Pull request by pinak22dhir**
pls have a look,now thanks

### CODE REVIEW

File: code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<h2>Code Of Conduct-CYBERSENTINEL</h2>
```
Comment (bhaskar1001101) [inline code_of_conduct.md]: I'm unable to make sense of the word CYBERSENTINEL. The rest looks good.

Comment (pinak22dhir) [reply code_of_conduct.md]: changed in file section @bhaskar1001101 or should i again make another branch

**Review by bhaskar1001101 — APPROVED** [review]
LGTM. Merging your pull request.

**Comment by pinak22dhir** [comment]
@bhaskar1001101 and @TeeWrath please add kwoc tag

**Pull request merged**